xarray/namedarray/utils.py

```diff
@@ -57,11 +118,17 @@ def is_duck_array(value: typing.Any) -> TypeGuard[T_DuckArray]:
     )
 
 
-def is_duck_dask_array(x: typing.Any) -> bool:
-    return is_duck_array(x) and is_dask_collection(x)
+def is_duck_dask_array(x: T_DuckArray) -> TypeGuard[DaskArray]:
+    return is_dask_collection(x)
+
+
+def is_chunked_duck_array(x: T_DuckArray | T_ChunkedArray) -> TypeGuard[T_ChunkedArray]:
```

Comment:
Ok I see now. You are not testing for the _Array properties that's why you expect the argument to be an _Array type already. Makes sense.

Reply:
The idea here is that I don't have to check `_Array` properties, because I trust that `x` is input as a `T_DuckArray` or the narrower `T_ChunkedArray`.